### PR TITLE
[Kotlin Migration] BasicImageFieldControllerTest

### PR DESCRIPTION
**Pull request by Akshay0701**

### CODE REVIEW

File: AnkiDroid/kotlinMigration.gradle [hunk 1]
```diff
@@ -42,8 +42,8 @@ permission notice:
 
 // Example of class name: "/com/ichi2/anki/UIUtils.kt"
 // Ensure that it starts with '/' (slash)
-def source = Source.MAIN
-def className = ""
+def source = Source.TEST
+def className = "com/ichi2/anki/multimediacard/fields/BasicImageFieldControllerTest.kt"
```
Comment (david-allison-1) [inline AnkiDroid/kotlinMigration.gradle]: `/`

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
LGTM otherwise

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Coming back to this: could we re-import `MatcherAssert, Matchers, Mockito`

Only nits though, happy regardless

**Review by david-allison-1 — APPROVED** [review]

**Review by ShridharGoel — APPROVED** [review]

**Comment by david-allison-1** [comment]
@ShridharGoel please rebase merge, rather than squash merge in future. 

We lose `git blame` otherwise.

**Comment by ShridharGoel** [comment]
@david-allison-1 Okay, thanks for letting me know.

Is there a way to fix it for this PR?

**Comment by Akshay0701** [comment]
yes but it's okay for this pr, because it is a test, and the chances of breaking tests is also less, so no worries for now

**Comment by david-allison-1** [comment]
Not without rebasing master

**Comment by mikehardy** [comment]
@ShridharGoel  Also when I merge PRs I add them to the next release milestone - which is what I use to make the changelogs for release. And of course, that's not documented anywhere ;-)

**Pull request merged**